info.json

```diff
@@ -32,14 +30,6 @@
             {
                 "name": "07 - ChatOps"
```

Comment:
As per new naming convention, 07 is 07 - Incident Response Plan --- (Move PBs from 09 - Incident Response)
Please move this to 09

Reply:
@cs-suyog-jinturkar Updated playbook collection sequence to 09